README.md

```diff
@@ -1,39 +1,58 @@
-# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=master)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
-Kubernetes deployment of RADAR-base.
+# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=dev)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
+The repository contains Helm charts and other configuration needed to install RADAR-base on a Kubernetes cluster.
 
+## Prerequisites
```

Comment:
Perhaps split this into an Infrastructure header and a Dependencies header. The former to describe the infrastructure, the latter to describe software that needs to be installed by the user.